app/views/course/assessment/question/text_responses/_text_response.json.jbuilder

```diff
@@ -26,7 +26,7 @@ json.groups question.groups.each do |group|
       # Sanitization will be handled automatically by the React frontend.
       json.solution s.solution.join(', ')
       json.solutionLemma s.solution_lemma.join(', ')
-      json.explanation s.explanation
+      json.explanation s.explanation.strip
```

Comment:
If we strip during the input I don't think we need to strip when showing the attributes.

Reply:
The `explanation` input is a summernote editor.

Reply:
But you stripped it [here](https://github.com/Coursemology/coursemology2/pull/2876/files#diff-1da16f5bad998c15c3330d3897225b1eR234) already right? Plus, if it is rich text, I'm not sure whether stripping is useful because it cannot catch trailing spaces within html tags.

Reply:
The `strip` in the autograding service is for the 'combined' hint that appears after the Wrong/Correct status, while the `strip` here is for the explanations that are shown beside each solution word in a tabular format.